pkg/login/social/connectors/social_base.go

```diff
@@ -36,17 +36,18 @@ type SocialBase struct {
 func newSocialBase(name string,
 	config *oauth2.Config,
 	info *social.OAuthInfo,
-	autoAssignOrgRole string,
 	features featuremgmt.FeatureToggles,
+	cfg *setting.Cfg,
 ) *SocialBase {
 	logger := log.New("oauth." + name)
 
 	return &SocialBase{
 		Config:            config,
 		info:              info,
 		log:               logger,
-		autoAssignOrgRole: autoAssignOrgRole,
+		autoAssignOrgRole: cfg.AutoAssignOrgRole,
```

Comment:
this one is not really needed anymore since we store `setting.Cfg` directly

Reply:
Thanks, done!